pkg/workflows/sdk/gen/compute.go.tmpl

```diff
@@ -6,6 +6,39 @@ import (
     "github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
+type ComputeConfig[C any] struct {
+    Config C
+}
+
+func (c *ComputeConfig[C]) ToMap() (map[string]any, error) {
+    wc, err := values.WrapMap(c.Config)
+    if err != nil {
+        return nil, err
+    }
+
+    uc, err := wc.Unwrap()
+    if err != nil {
+        return nil, err
+    }
+
+    cm, ok := uc.(map[string]any)
+    if !ok {
+        return nil, errors.New("could not convert config into map")
+    }
+
+    if _, ok := cm["config"]; ok {
+        return nil, errors.New("`config` is a reserved keyword inside Compute config")
+    }
+    cm["config"] = "$(ENV.config)"
+
+    if _, ok := cm["binary"]; ok {
+        return nil, errors.New("`binary` is a reserved keyword inside Compute config")
+    }
+    cm["binary"] = "$(ENV.binary)"
+
+    return cm, nil
```

Comment:
do we need a check for outputs too?

Reply:
I'm not sure :/ Can you say more about why you think we need it?